packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -50,6 +56,12 @@ export class InteractionTracker {
     [index: string]: number;
   } = {};
 
+  // Map intended to keep track of current XHR objects
+  // associated to a span.
+  private readonly xhrTasks: {
+    [index: string]: XMLHttpRequest;
```

Comment:
Instead of `index` could you call this `spanId`? Could you also explain that we use object equality to match XHR instances to spans but that an object can only accept string or number keys so the mapping must be from span ID to XHR?

Reply:
An alternative would be to use a `Map`, which is only available in newer versions of JavaScript (old versions are polyfilled), but which allows setting objects as the keys.

Reply:
I changed this as `Map` as it's more readable and easier to understand.